tools/firely-server-import.rst

```diff
@@ -0,0 +1,182 @@
+      "indexBufferSize": 50
+    }
+  }
+
+Supported arguments
+^^^^^^^^^^^^^^^^^^^
+
++-------------------------------------------+----------------------------------+----------+--------------------------------------------------------------------------------------------------------------------------------------------+
+| CLI argument                              | Appsettings parameter name       | Required | Description                                                                                                                                |
++===========================================+==================================+==========+============================================================================================================================================+
+| -f, --fhir-version <R3|R4>                | fhirVersion                      |          | FHIR version of the input                                                                                                                  |
++-------------------------------------------+----------------------------------+----------+--------------------------------------------------------------------------------------------------------------------------------------------+
+| -s, --source <source>                     | source                           | yes      | Input directory for work                                                                                                                   |
++-------------------------------------------+----------------------------------+----------+--------------------------------------------------------------------------------------------------------------------------------------------+
+| -l, --limit <limit>                       | limit                            |          | Limit the number of resources to import. Use this for testing your setup.                                                                  |
++-------------------------------------------+----------------------------------+----------+--------------------------------------------------------------------------------------------------------------------------------------------+
+| --license <license>                       | license                          | yes      | Firely Server license file                                                                                                                 |
++-------------------------------------------+----------------------------------+----------+--------------------------------------------------------------------------------------------------------------------------------------------+
+| --update-existing-resources <true|false>  | updateExistingResources          |          | When true, a resource is updated in the database if it already exists. Otherwise a resource is updated and history is created.             |
```

Comment:
I thought 'false' means that the new resource is ignored.
See Orchestration.cs, `filterExistingBlock`.

Reply:
Does this make more sense: `
When true, a resource is updated in the database if it already exists and a history record is created. Otherwise, an existing resource gets skipped.`?